cosv-backend/build.gradle.kts

```diff
@@ -14,4 +14,8 @@ dependencies {
     implementation(libs.spring.security.core)
     implementation(libs.spring.data.jpa)
     implementation(libs.hibernate.jpa21.api)
+
+    //needed for kubernetes configs and secrets to be added as additional configurations
+
+    implementation(libs.spring.cloud.starter.kubernetes.client.config)
```

Comment:
```suggestion
    implementation(libs.spring.cloud.starter.kubernetes.client.config) {
        because("needed for kubernetes configs and secrets to be added as additional configurations")
    }
```

Reply:
and please add the same in existed dependency